crates/astria-utils/src/config.rs

```diff
@@ -0,0 +1,17 @@
+use figment::{
+    providers::Env as FigmentEnv,
+    Figment,
+};
+use serde::{
+    Deserialize,
+    Serialize,
+};
+
+pub trait AstriaConfig<'a>: Serialize + Deserialize<'a> {
```

Comment:
Can you make the prefix an associated const of this trait?

You can have:
```rust
pub trait AstriaConfig<'a>: Serialize + Deserialize<'a> {
    const PREFIX: &'static str;
```
And then in the macro you can set that prefix from the proc macro (or preferably derive macro).